composer.json

```diff
@@ -67,7 +67,8 @@
         "phpstan/phpstan-nette": "^0.6",
         "phpstan/phpstan-doctrine": "^0.6",
         "object-calisthenics/phpcs-calisthenics-rules": "v3.0.0-RC3",
-        "roave/security-advisories": "dev-master"
+        "roave/security-advisories": "dev-master",
+        "gecko-packages/gecko-php-unit": "^2.0"
```

Comment:
it's there for 6 months:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blame/2.2/composer.json#L34